dataprep/eda/missing/compute.py

```diff
@@ -133,13 +134,26 @@ def missing_impact_1vn(  # pylint: disable=too-many-locals
             histogram(df[col], bins=bins, return_edges=True, range=range)
             for df in [df0, df1]
         ]
-    (hists,) = dd.compute(hists)
+
+        # In some cases(Issue#98), dd.compute() can change the features dtypes and cause error.
+        # So we need to restore features dtypes after dd.compute()。
+        centers_dtypes = (hists[col][0][1].dtype, hists[col][1][1].dtype)
+        (hists,) = dd.compute(hists)
+        dict_value = []
+
+        # Here we have to avoid reusing hists, as dd.compute can change type again,
```

Comment:
Here we do not reassign to the "hists" variable, as ...,

Reply:
This comment is so confusing to anyone who doesn't have the context.

Reply:
Okay, thanks for pointing out. I have updated the comments.